src/data/global-data.ts

```diff
@@ -40,4 +40,5 @@ export const ContextPages = {
   
   Faq: ['faq'],
   howItWorks: ['howItWorks'],
+  termsOfUse: ['termsOfUse'],
```

Comment:
Needs to be written in pascal case, like another ContextPages

Reply:
These changes will be made in another branch